packages/apple-pay-integration/types/ApplePaySession.d.ts

```diff
@@ -0,0 +1,7 @@
+export {};
```

Comment:
Is there a reason for this? Could you do `export declare global` instead if you want to export something?

Reply:
Good point, I can update it